src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/Winmeta.cs

```diff
@@ -63,7 +63,15 @@ public enum EventTask
         /// <summary>
         /// Undefined task
         /// </summary>
-        None = 0
+        None = 0,
+        /// <summary>
+        /// Thread pool worker thread task
+        /// </summary>
+        ThreadPoolWorkerThread = 16,
```

Comment:
These are public types. I do not think we want to be adding these constants to these public types. 

All additions to public types need to go through the API review (https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/api-review-process.md).